apps/language_models/src/pipelines/falcon_pipeline.py

```diff
@@ -167,7 +167,7 @@ def compile_falcon(self):
                 ts_graph = import_with_fx(
                     model,
                     falconCompileInput,
-                    is_f16=self.precision == "fp16",
+                    precision=self.precision,
```

Comment:
I think making changes in all the existing working models/pipelines won't be a good move especially if you haven't tested this patch for all the related models/pipelines which are changed. We shouldn't merge this patch for now till these pipelines are tested or we are sure about whether all of these changes are required.

Reply:
I could just easily adding a precision arg for import_with_fx without changeing is_f16. Dan said we should probably get rid of the is_f16 arg if adding `precision`. I can revert it if you think it's better for merging this patch. I think we should merge this patch soon to unblock the int8 work.

Reply:
The precision arg is already there you can make use of that. And that would also eliminate the need for modifying the other models.

Reply:
there was no precision arg in import_with_fx

Reply:
reverted and added a todo for getting rid of is_f16